snap/snapcraft.yaml

```diff
@@ -0,0 +1,138 @@
+name: mycroft-precise
+adopt-info: mycroft-precise
+summary: Mycroft Precise
+description: Mycroft Precise, a wake word spotter.
+base: core18
+grade: stable
+confinement: strict
+build-packages: [python3-dev, python3-setuptools, libtool, libffi-dev, libssl-dev, autoconf, build-essential]
+parts:
+  alsa-mixin:
+    after: []
+    build-packages:
+    - libasound2-dev
+    plugin: dump
+    source: https://github.com/diddlesnaps/snapcraft-alsa.git
+    source-subdir: snapcraft-assets
+    stage-packages:
+    - libslang2
+    - libasound2
+    - libasound2-plugins
+    - libportaudio2
+    - pulseaudio
+    - alsa-utils
+  mycroft-precise:
+    after: [alsa-mixin]
+    plugin: python
+    python-version: python3
+    source: https://github.com/MycroftAI/mycroft-precise.git
+    source-type: git
+    source-branch: dev
```

Comment:
I wonder if we should fix up the branches before we publish this, and by default source this from the future master branch.

Reply:
It should be from the dev to work as both development/unstable and release versions

When a release is triggered dev will either match the latest tag -> a stable version string is generated
or
dev is different from the latest tag -> a version with "-dev" appendix will be added to the version.

Reply:
Dropped this section